<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LafdupWindow</class>
 <widget class="QWidget" name="LafdupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络剪切板</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="lstActions">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>发送内容</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="txtContent"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSend">
          <property name="text">
           <string>发送[Ctrl+Enter]</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>配置信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblInformation">
          <property name="text">
           <string>电脑IP:
192.168.199.2
10.1.1.2</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnManageKnownPeer">
            <property name="text">
             <string>设置手机&amp;IP...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetPassword">
            <property name="text">
             <string>设置密码(&amp;P)...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
